### PR TITLE
Allow chevrons to be hidden

### PR DESCRIPTION
**Pull request by shamilovtim**
Simple bool to allow the chevrons to be hidden

### CODE REVIEW

File: lib/src/calendar.dart [hunk 1]
```diff
@@ -372,12 +373,13 @@ class _TableCalendarState extends State<TableCalendar> with SingleTickerProvider
...
+      if (widget.headerStyle.showRightChevron)
+        _CustomIconButton(
+          icon: widget.headerStyle.rightChevronIcon,
+          onTap: _selectNext,
+          margin: widget.headerStyle.rightChevronMargin,
+          padding: widget.headerStyle.rightChevronPadding,
+        ),
     ];
 
     if (widget.headerStyle.formatButtonVisible && widget.availableCalendarFormats.length > 1) {
```
Comment (aleksanderwozniak) [inline lib/src/calendar.dart:385]: This change breaks the calendar here

**Comment by shamilovtim** [comment]
You mean cause of the merge conflict, right? I'll take a look.

**Comment by shamilovtim** [comment]
fixed merge conflict

**Comment by aleksanderwozniak** [comment]
No no, I meant the code where `FormatButton` gets inserted into `children` at position 2(3), but it can't be inserted there when `children` contains only one item (which happens when you hide chevrons).

**Comment by shamilovtim** [comment]
fixed bug

**Pull request merged**